utils/report.c

```diff
@@ -178,7 +182,8 @@ static struct sort_key sort_func = {
 
 static struct sort_key *all_sort_keys[] = {
 	&sort_total,	&sort_total_avg, &sort_total_min, &sort_total_max, &sort_self,
-	&sort_self_avg, &sort_self_min,	 &sort_self_max,  &sort_call,	   &sort_func,
+	&sort_self_avg, &sort_self_min,	 &sort_self_max,  &sort_call,	   &sort_caller,
+	&sort_callee,	&sort_depth_min, &sort_depth_max, &sort_func,
```

Comment:
I'm not sure pre-commit / clang-format works correctly.

Reply:
I passed the clang-format above. Could you tell me what the problem is?

Reply:
Oh, nevermind.  I concerned about the indentation as if `clang-format` didn't run but it looks github displays the indentation a little bit different than in a terminal.